### PR TITLE
[DPE-5364] Block Kyuubi when zookeeper not integrated for n>1

### PR DESCRIPTION
**Pull request by theoctober19th**

### CODE REVIEW

**Review by deusebio — APPROVED** [review]
Looks very nice!

File: src/events/kyuubi.py [hunk 1]
```diff
@@ -70,3 +78,19 @@ def _on_kyuubi_pebble_ready(self, event: ops.PebbleReadyEvent):
...
             zookeeper_info=self.context.zookeeper,
         )
+
+    @compute_status
+    def _on_peer_relation_joined(self, event: ops.RelationJoinedEvent):
+        """Handle the peer relation joined event.
+
+        This is necessary for updating status of all units upon scaling up/down.
+        """
+        pass
```
Comment (welpaolo) [inline src/events/kyuubi.py:88]: Should we log the event (debug level) ?

File: src/events/kyuubi.py [hunk 2]
```diff
@@ -70,3 +78,19 @@ def _on_kyuubi_pebble_ready(self, event: ops.PebbleReadyEvent):
...
+
+    @compute_status
+    def _on_peer_relation_departed(self, event: ops.RelationDepartedEvent):
+        """Handle the peer relation departed event.
+
+        This is necessary for updating status of all units upon scaling up/down.
+        """
+        pass
```
Comment (welpaolo) [inline src/events/kyuubi.py:96]: Same here?

**Review by welpaolo — APPROVED** [review]
Looks good to me. 
I left a minor comment

File: src/events/kyuubi.py (hunk 2 above)
Comment (theoctober19th) [reply src/events/kyuubi.py]: Done

File: src/events/kyuubi.py (hunk 1 above)
Comment (theoctober19th) [reply src/events/kyuubi.py]: Done

**Pull request merged**